docs/02-app/02-api-reference/04-functions/cacheLife.mdx

```diff
@@ -0,0 +1,249 @@
+
+> **Good to know**: The “stale” property does not set the `Cache-control: max-age` header. It instead controls the client-side router cache.
+
+## Examples
+
+### Defining reusable cache profiles
+
+You can create a reusable cache profile by defining them in your `next.config.ts` file. Choose a name that suits your use case and set values for the `stale`, `revalidate`, and `expire` properties. You can create as many custom cache profiles as needed. Each profile can be referenced by its name as a string value passed to the `cacheLife` function.
+
+```ts filename="next.config.ts" switcher
+import type { NextConfig } from 'next'
+
+const nextConfig: NextConfig = {
+  experimental: {
+    dynamicIO: true,
+    cacheLife: {
+      biweekly: {
+        stale: 60 * 60 * 24 * 14, // 14 days
+        revalidate: 60 * 60 * 24, // 1 day
+        expire: 60 * 60 * 24 * 14, // 14 days
+      },
+    },
+  },
+}
+
+module.exports = nextConfig
+```
+
+```js filename="next.config.js" switcher
+const nextConfig = {
+  experimental: {
+    dynamicIO: true,
+    cacheLife: {
+      biweekly: {
+        stale: 60 * 60 * 24 * 14, // 14 days
+        revalidate: 60 * 60 * 24, // 1 day
+        expire: 60 * 60 * 24 * 14, // 14 days
+      },
+    },
+  },
+}
+
+module.exports = nextConfig
+```
+
+The example above caches for 14 days, checks for updates daily, and expires the cache after 14 days. You can then reference this profile throughout your application by its name:
+
+```tsx filename="app/page.tsx" highlight={4}
+'use cache'
+import { unstable_cacheLife as cacheLife } from 'next/cache'
+
+export default async function Page() {
+  cacheLife('biweekly')
+  return <div>Page</div>
+}
+```
+
+### Overriding the default cache profiles
+
+While the default cache profiles provide a useful way to think about how fresh or stale any given part of cacheable output can be, you may prefer different named profiles to better align with your applications caching strategies.
+
+You can override the default named cache profiles by creating a new configuration with the same name as the defaults.
+
+The example below shows how to override the default “days” cache profile:
+
+```ts filename="next.config.ts"
+const nextConfig = {
+  experimental: {
+    dynamicIO: true,
+    cacheLife: {
+      days: {
+        stale: 3600, // 1 hour
+        revalidate: 900, // 15 minutes
+        expire: 86400, // 1 day
+      },
+    },
+  },
+}
+
+module.exports = nextConfig
+```
+
+### Defining cache profiles inline
+
+For specific use cases, you can set a custom cache profile by passing an object to the `cacheLife` function:
+
+```tsx filename="app/page.tsx" highlight={5,6,7}
+'use cache'
+import { unstable_cacheLife as cacheLife } from 'next/cache'
+
+cacheLife({
```

Comment:
Given that you cannot actually call the function outside the scope of a function, should we put this example within a function to avoid any unintended misunderstandings?

Reply:
Good catch, will update!